metasyn/distribution/base.py

```diff
@@ -28,8 +29,14 @@ class attributes of a distribution.
 class BaseDistribution(ABC):
     """Abstract base class to define a distribution.
 
-    All distributions should be derived from this class, and should implement the following methods:
-    `_fit`, `draw`, `_param_dict`, `_param_schema`, `default_distribution` and `__init__`.
+    All distributions should be derived from this class, and should implement
+    the following methods:
+    :meth:`~metasyn.distribution.base.BaseDistribution._fit`,
```

Comment:
From what I read, you can also do relative paths, maybe that will be slightly easier to read at least in the code.

Reply:
I actually think this reads nicer than:

```
    :meth:`~_fit`,
    :meth:`~draw`,
    :meth:`~_param_dict`,
    :meth:`~_param_schema`,
    :meth:`~default_distribution`
    and ``__init__``.
    
  ```
  
  In this context, do you have a strong preference against this?

Reply:
I do think it's a bit nicer yeah, might be able to fit them on the same line. Btw, I saw that some of these distributions are starting with an underscore and don't show up in the docs. Maybe it would be good for this class to show at least the abstract underscore classes and improve the docstrings on the a bit, because they're really short right now.

Reply:
Yeah those disabled links and lacking documentation is bothering me too. I would love to show it, but I am a bit confused. How do you mean showing them?

Reply:
Yes, they are showing now in the documentation, thanks!